@kiva/kv-shop/src/shopError.ts

```diff
@@ -35,6 +35,12 @@ export function parseShopError(error: any) {
 			original: error,
 		}, 'There was a problem validating your payment information. Please double-check the details and try again.');
 	}
+	if (errorCode === 'shop.invalidBasketId' || errorCode === 'shop.basketRequired') {
+		return new ShopError({
+			code: errorCode,
+			original: error,
+		}, 'There was a problem with your donation, refreshing the page to try again.');
```

Comment:
I don't think this is an appropriate message for all instances of these error codes. I think we should pass in a `context` param to this method that can be used to select context specific error messages.